Crash when opening the preferences

When opening the preferences, the application crashes.

Device: Moto G5, running Android 7.0

```
Shutting down VM
FATAL EXCEPTION: main
Process: com.github.nutomic.controldlna, PID: 28483
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.github.nutomic.controldlna/com.github.nutomic.controldlna.gui.PreferencesActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.app.ActionBar.setDisplayHomeAsUpEnabled(boolean)' on a null object reference
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2659)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2724)
       at android.app.ActivityThread.-wrap12(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1473)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6123)
       at java.lang.reflect.Method.invoke(Native Method)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:867)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:757)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.app.ActionBar.setDisplayHomeAsUpEnabled(boolean)' on a null object reference
       at com.github.nutomic.controldlna.gui.PreferencesActivity.onCreate(PreferencesActivity.java:70)
       at android.app.Activity.performCreate(Activity.java:6672)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1140)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2612)
       ... 9 more

```

--- a/src/main/java/com/github/nutomic/controldlna/gui/PreferencesActivity.java
+++ b/src/main/java/com/github/nutomic/controldlna/gui/PreferencesActivity.java
@@ -66,7 +66,8 @@ public class PreferencesActivity extends PreferenceActivity
 		// There is currently no way to get ActionBar in PreferenceActivity on pre-honeycomb with
 		// compatibility library, so we'll have to do a version check.
 		if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.HONEYCOMB) {
-			getActionBar().setDisplayHomeAsUpEnabled(true);
+// crashes in Android 7.0:
+//			getActionBar().setDisplayHomeAsUpEnabled(true);
 		}
 
 		PreferenceManager.getDefaultSharedPreferences(this)

--- a/src/main/java/com/github/nutomic/controldlna/gui/PreferencesActivity.java
+++ b/src/main/java/com/github/nutomic/controldlna/gui/PreferencesActivity.java
@@ -63,12 +63,9 @@ public class PreferencesActivity extends PreferenceActivity
 	public void onCreate(Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
 
-		// There is currently no way to get ActionBar in PreferenceActivity on pre-honeycomb with
-		// compatibility library, so we'll have to do a version check.
-		if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.HONEYCOMB) {
-// crashes in Android 7.0:
-//			getActionBar().setDisplayHomeAsUpEnabled(true);
-		}
+		// Ideally this code should call getActionBar().setDisplayHomeAsUpEnabled(true) but
+		// it isn't possible to do that with the current appcompat library without re-writing
+		// preferences to use fragments
 
 		PreferenceManager.getDefaultSharedPreferences(this)
 				.registerOnSharedPreferenceChangeListener(this);

